misc/netgen/remove-x11-requirement.patch

```diff
@@ -0,0 +1,38 @@
+diff --git a/scripts/configure b/scripts/configure
```

Comment:
We should send this upstream to fix https://github.com/RTimothyEdwards/netgen/issues/9 at some point :)

Reply:
I'm waiting to see if it actually builds.

It's entirely possible that it won't actually build, and that it's relying on include files that are leaking in from my environment. However, I can say for sure that it's building on my system with this patch, and without this patch it errored with trying to find libX11.

We'll need to see what CI says.